packages/react-sdk/src/components/Shortcuts/ClipboardShortcuts/ClipboardShortcuts.tsx

```diff
@@ -147,11 +153,44 @@ export function ClipboardShortcuts() {
         const pastedElementIds = slideInstance.addElements(elements);
         slideInstance.setActiveElementIds(pastedElementIds);
       }
+
+      // Handle image files
+      const files = event.clipboardData?.files;
```

Comment:
Please reuse the `ImageUploadContext` here. It does all the checks already. Otherwise we duplicate code.

Also see https://github.com/nordeck/matrix-neoboard/blob/5e8e280ce82fddadc51f2c312910d3d388909b50/packages/react-sdk/src/components/ImageUpload/useSlideImageUpload.tsx#L55-L63

Maybe we should pull the handling out of `useSlideImageUpload`  into a reusable function.

Reply:
I now resued the code to scale and center the image.